addons/website/models/ir_http.py

```diff
@@ -82,6 +83,10 @@ def _slug_matching(cls, adapter, endpoint, **kw):
 
     @classmethod
     def _generate_routing_rules(cls, modules, converters):
+        if not request:
```

Comment:
How is it that at this level there is no request object?

Reply:
When you call `website.search_pages` via RPC, at some point you'll generate the routing-map